ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/router-internet-exit.md

```diff
@@ -0,0 +1,55 @@
+    | [<samp>&nbsp;&nbsp;exit_groups</samp>](## "router_internet_exit.exit_groups") | List, items: Dictionary |  |  |  | Exit groups represent a group of exit options (connections).<br>Traffic flows are load balanced in a round robin fashion across all the members (exits) of the exit-group. |
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;name</samp>](## "router_internet_exit.exit_groups.[].name") | String | Required, Unique |  |  |  |
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;fib_default</samp>](## "router_internet_exit.exit_groups.[].fib_default") | Boolean |  |  |  | Fib default exit indicates that the flows that select this exit will follow the default route available in the VRF of the flow. |
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;locals</samp>](## "router_internet_exit.exit_groups.[].locals") | List, items: Dictionary |  |  |  | Local connection refers to a connection configured under the service-insertion CLI mode.<br>The service-insertion module reports the health of the connection and the exit will qualify for use only when it is healthy. |
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;connection</samp>](## "router_internet_exit.exit_groups.[].locals.[].connection") | String |  |  |  |  |
+
+=== "YAML"
+
+    ```yaml
+    # Internet-exit feature provides internet bound service offered in an AVT to the traffic going outside the enterprise towards the internet.
+    router_internet_exit:
+
+      # Internet-exit policy represent a policy which can be attached to an AVT profile that needs to be applied to the traffic matching the AVT.
+      policies:
+        - name: <str; required; unique>
+
+          # The exit groups that are configured under a policy are strictly ordered, meaning an exit group appearing first has more priority than the exit group that follows it.
+          exit_groups:
+            - name: <str; required; unique>
+
+              # Insert the previous exit group AFTER the following exit group.
+              after: <str>
+
+              # Insert the previous exit group BEFORE the following exit group.
+              before: <str>
+
+      # Exit groups represent a group of exit options (connections).
+      # Traffic flows are load balanced in a round robin fashion across all the members (exits) of the exit-group.
+      exit_groups:
+        - name: <str; required; unique>
+
+          # Fib default exit indicates that the flows that select this exit will follow the default route available in the VRF of the flow.
+          fib_default: <bool>
+
+          # Local connection refers to a connection configured under the service-insertion CLI mode.
+          # The service-insertion module reports the health of the connection and the exit will qualify for use only when it is healthy.
+          locals:
+            - connection: <str>
```

Comment:
```suggestion
          local_connections:
            - <str>
            - <str>
```
just a suggestion

Reply:
agreeing with the name - though lets do

```
          local_connections:
	            - name: <str>
	            - name: <str>
```

to cater for possible future growth

Reply:
Done!